app/views/granules/prepare_script.html.erb

```diff
@@ -55,6 +55,7 @@ netrc=$(mktemp netrc.XXXXXXXXXX)
 chmod 0600 "$cookiejar" "$netrc"
 function finish {
   rm -rf "$cookiejar" "$netrc"
+  <% if @first_link %>
   <% if @first_link.start_with? 'ftp' %>
```

Comment:
I'll leave it to @mightynimble to comment on whether or not this is the best way to add a check... but if it is, I think we should indent the nested `if` statement for readability. So just indent lines 59-67 by one (and also the other nested ifs added by this PR) and this is good to go.